src/sensor_state_data/device_trigger/event_device_keys.py

```diff
@@ -0,0 +1,16 @@
+# Supported devices keys for devices that fire events
+
+from ..enum import StrEnum
+
+
+class EventDeviceKeys(StrEnum):
```

Comment:
Is there a reason why we would need to go this far with defining a data model? Why can't we have the integration just listen for bluetooth packets that are marked as "event" by the sensor-state-data ?

Reply:
Removed it, also the event_types file, for the same reason.